docs/Using Fleet/Usage-statistics.md

```diff
@@ -25,6 +25,10 @@ Below is the JSON payload that is sent to Fleet Device Management Inc:
   "hostExpiryEnabled": true,
   "mdmWindowsEnabled": false,
   "liveQueryDisabled": false,
+  "aiFeaturesDisabled": false,
+  "maintenanceWindowsConfigured": true,
```

Comment:
### Dev note

Maintenance windows are considered "configured" if:

+ configuration has value set for `integrations.google_calendar[0].domain`
+ configuration has value set for `integrations.google_calendar[0].api_key_json`